packages/markitdown/src/markitdown/converters/_markdownify.py

```diff
@@ -87,5 +87,14 @@ def convert_img(self, el: Any, text: str, convert_as_inline: bool) -> str:
 
         return "![%s](%s%s)" % (alt, src, title_part)
 
+    def convert_em(self, el: Any, text: str, convert_as_inline: bool) -> str:
```

Comment:
noticed that it doesn't have an <em> tag, and that's used in Epub as far as I know